src/sst/core/interprocess/mmapparent.cc

```diff
@@ -24,8 +24,8 @@ template class SST::Core::Interprocess::MMAPParent<testtunnel>;
 namespace SST::Core::Interprocess {
 
 int
-SST_MPI_Comm_spawn_multiple(int count, char* array_of_commands[], char** array_of_argv[], const int array_of_maxprocs[],
-    const char* array_of_env[])
+SST_MPI_Comm_spawn_multiple(int UNUSED(count), char* UNUSED(array_of_commands)[], char** UNUSED(array_of_argv)[],
```

Comment:
If these only show up for builds that use --disable-mpi, then the proper macro to use is UNUSED_WO_MPI (defined in sst_mpi.h)